.travis.yml

```diff
@@ -79,6 +79,6 @@ jobs:
       if: type = push AND branch =~ /^release-[0-9]+\..*$/
       script:
         - make pipeline-manifest/update PIPELINE_MANIFEST_COMPONENT_SHA256=${TRAVIS_COMMIT} PIPELINE_MANIFEST_COMPONENT_REPO=${TRAVIS_REPO_SLUG} PIPELINE_MANIFEST_BRANCH=${TRAVIS_BRANCH}
-        - sleep 60
+        - sleep 300
```

Comment:
Curious about how we could accomplish this without a hard-coded sleep. However, this solution should accomplish the goal

Reply:
Yeah, I talked with Gurney about this and there seems to be no good way right now to wait and detect when the pipeline-manifest/update job completes to then trigger the next one for the test image.